training-service/src/app/content/course/component/course-list/course-list.component.ts

```diff
@@ -19,12 +23,14 @@ export class CourseListComponent implements OnInit {
     ])
   });
 
-  course: Course[] = [];
-
   constructor(private courseService: CourseService) { }
 
-  ngOnInit(): void{
+  ngOnInit() {
     this.course = this.courseService.getCourseList();
   }
 
+  convertDurationToTime(item: number):any {
```

Comment:
please add proper return type and convert to arrow function

Reply:
done